_data/simple-icons.json

```diff
@@ -1800,6 +1800,12 @@
             "hex": "CC092F",
             "source": "https://en.wikipedia.org/wiki/Broadcom_Inc"
         },
+        {
+            "title": "BSD",
+            "hex": "AB2B28",
+            "source": "https://freebsdfoundation.org/about-us/about-the-foundation/project",
+            "guidelines": "https://freebsdfoundation.org/about-us/about-the-foundation/project"
```

Comment:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:BSD_wordmark.svg"
```
I'm not 100% about using `FreeBSD` as our source, given it's one of a few projects that originate the BSD license. For that sake, we're probably safer using the Wikimedia source.

Reply:
The wikimedia icon is from the same source
https://commons.wikimedia.org/wiki/File:BSD_wordmark.svg
![image](https://github.com/simple-icons/simple-icons/assets/35000807/1acc4e48-9947-4c69-b0ea-e970295efad6)

Reply:
Ignore me then!